aws/scenarios/microVMs/config/config.go

```diff
@@ -46,3 +49,30 @@ func (e *DDMicroVMConfig) GetStringWithDefault(config *sdkconfig.Config, paramNa
 
 	return defaultValue
 }
+
+func (e *DDMicroVMConfig) GetIntWithDefault(config *sdkconfig.Config, paramName string, defaultValue int) int {
```

Comment:
You should probably remove the duplication and get `aws.Environment` in `NewMicroVMConfig` instead of `pulumi.Context`.

Reply:
Does this address your point?
https://github.com/DataDog/test-infra-definitions/pull/51/commits/56e563bd962baa5590da0553943b6678bd7e53ba

Reply:
Yes but then the goal is to remove all the `GetXXXWithDefault` functions that you copied and use `e.Get...` directly

Reply:
Ah Gotcha